x-pack/plugin/esql/qa/testFixtures/src/main/resources/k8s-metrics.csv-spec

```diff
@@ -1,6 +1,6 @@
 metricsWithoutAggs
-required_capability: metrics_syntax
-METRICS k8s | sort @timestamp DESC, cluster, pod | keep @timestamp,cluster,pod,network.bytes_in,network.cost | limit 5;
+required_capability: rate_aggregation
+FROM k8s | sort @timestamp DESC, cluster, pod | keep @timestamp,cluster,pod,network.bytes_in,network.cost | limit 5;
```

Comment:
I thought we wanted to keep `METRICS k8s | ...` when propagating the `_tsid` and preserving the sort ordering?